app/views/users/new.html.haml

```diff
@@ -1,7 +1,13 @@
+= content_for :head do 
+  %script{:src => "https://js.stripe.com/v2/", :type => "text/javascript"}
+  :javascript
+    Stripe.setPublishableKey('pk_test_XLSZT1qESCJZ6NHJ3L7ms3Nm');
```

Comment:
I see you actually set it in an env so you can use it here `Stripe.setPublishableKey("#{ENV['PUBLISHABLE_KEY']}")`